cmd/osv-scanner/main_test.go

```diff
@@ -1520,6 +1520,48 @@ Filtered 2 vulnerabilities from output
 	}
 }
 
+func TestRun_SubCommands(t *testing.T) {
```

Comment:
maybe also add some comment that we should test other sub-commands as well if there will be more in the future

Reply:
Good suggestion. Thanks!